style-guide.md

```diff
@@ -0,0 +1,112 @@
+# Dockstore Docs Style Guide
+
+This style guide is based upon [Google's style guide](https://developers.google.com/style/) but contains several sections specific to Dockstore itself. It is not exhaustive nor authoritative; this document is designed as a quick reference guide.
+
+## Accessibility
+See also: https://developers.google.com/style/accessibility
+
+* Always set descriptive alt text for images, except for decorative images
+ 	* If the image contains important text, repeat that text in the alt text
+	 	* Although it is tricky due to how documentation websites tend to use images to clarify written text,  try to avoid having alt text repeat what is in the body text
+	 	* Do not start with "image of" or "picture of" but do describe the type of image, like:
+	 		* "Headshot of David Haussler smiling"
+		 	* "Screenshot of the UI of dockstore.org/my-tools with the Refresh Organization button circled"
+		 	* "Animation of a tabby cat jumping three feet into the air upon noticing a cucumber"
+* Keep in mind ~5% of the population is red-green colorblind
+	* 	When color schemes convey non-essential meaning, avoid using color schemes that would be inaccessible to red-green colorblind users
+	* 	When color schemes convey essential meaning (vital UI elements, etc) they should be functional in grayscale
+* Do not use things that would be a nightmare for screen readers
+ 	* Avoid using non-ASCII symbols 𝙨𝙪𝙘𝙝 𝙖𝙨 𝙩𝙝𝙞𝙨, 𝙬𝙝𝙞𝙘𝙝 𝙪𝙨𝙚𝙨 𝙘𝙤𝙙𝙚𝙥𝙤𝙞𝙣𝙩𝙨 𝙙𝙚𝙨𝙞𝙜𝙣𝙚𝙙 𝙛𝙤𝙧 𝙢𝙖𝙩𝙝𝙚𝙢𝙖𝙩𝙞𝙘𝙨, 𝙣𝙤𝙩 𝙩𝙚𝙭𝙩
```

Comment:
such as what (might be import issue)

Reply:
Good idea, clarified in https://github.com/dockstore/dockstore-documentation/pull/197/commits/a99b247e8fcabda289aac18c811314482e02f06f

Reply:
We can still use ye 𝖔𝖑𝖉𝖊 𝖒𝖊𝖙𝖆𝖑 𝖋𝖔𝖓𝖙 in PRs right?  lol